be/src/exec/pipeline/pipeline_driver_queue.cpp

```diff
@@ -110,6 +111,10 @@ size_t QuerySharedDriverQueue::size() {
 
 void QuerySharedDriverQueue::update_statistics(const DriverRawPtr driver) {
     _queues[driver->get_driver_queue_level()].update_accu_time(driver);
+    if (SourceOperator* source_operator = driver->source_operator()) {
```

Comment:
`QuerySharedDriverQueue::update_statistics` is used to update statistics for the driver queue. Maybe this logic is better to move to `PipelineDriver::_update_statistics`? In this way, we needn't write duplicated codes twice in `QuerySharedDriverQueue::update_statistics` and `DriverQueueWithWorkGroup::update_statistics`.

Reply:
Good advice, I could move it the `PipelineDriver::process` and `PipelineDriver::_update_statistics`.